documentation/modules/ROOT/pages/configuration/avro.adoc

```diff
@@ -199,3 +358,5 @@ Note that some details around Kafka Connect converters have slightly changed sin
 
 For a complete example of using Avro as the message format for {prodname} data change events,
 please see the https://github.com/debezium/debezium-examples/tree/master/tutorial#using-mysql-and-the-avro-message-format[MySQL and the Avro message format] tutorial example.
+
+endif::[community]
```

Comment:
Looks like this is a typo, should be `endif::community[]`?

Reply:
Yes, that's a typo. Fixed in the next commit.